stacks/tests/org.jboss.tools.stacks.core.test/src/org/jboss/tools/stacks/core/test/StacksCoreTest.java

```diff
@@ -10,23 +10,12 @@
  ******************************************************************************/
 package org.jboss.tools.stacks.core.test;
 
-/*************************************************************************************
- * Copyright (c) 2013 Red Hat, Inc. and others.
```

Comment:
Generally we add dates, not replace them, IIRC. So (c) 2013, 2018 or 2013-2018 is better here. @jeffmaury WDYT?